app/config/index.js

```diff
@@ -0,0 +1,52 @@
+const joi = require('joi')
+
+// Define config schema
+const schema = joi.object({
+  serviceName: joi.string().default('SFI prototype application'),
+  port: joi.number().default(3000),
+  env: joi.string().valid('development', 'test', 'production').default('development'),
+  staticCacheTimeoutMillis: joi.number().default(7 * 24 * 60 * 60 * 1000),
+  googleTagManagerKey: joi.string().default(''),
+  cookieOptions: joi.object({
+    ttl: joi.number().default(1000 * 60 * 60 * 24 * 365),
+    encoding: joi.string().valid('base64json').default('base64json'),
+    isSecure: joi.bool().default(true),
+    isHttpOnly: joi.bool().default(true),
+    clearInvalid: joi.bool().default(false),
+    strictHeader: joi.bool().default(true)
```

Comment:
There should also be a `isSameSite: joi.string().valid('Lax').default('Lax'),` property in here.